src/meta/config

```diff
@@ -0,0 +1,13 @@
+ngx_module_type=CORE
+ngx_module_name=ngx_meta_lua_module
+ngx_module_incs="$ngx_addon_dir"
+ngx_module_deps="$ngx_addon_dir/ddebug.h \
+                 $ngx_addon_dir/ngx_meta_lua_api.h \
+                 $ngx_addon_dir/ngx_meta_lua_module.h \
+                 $ngx_addon_dir/ngx_meta_lua_shdict.h"
+ngx_module_srcs="$ngx_addon_dir/ngx_meta_lua_module.c \
+                 $ngx_addon_dir/ngx_meta_lua_shdict.c"
+ngx_module_libs=
+ngx_module_link=ADDON
```

Comment:
This variable is set by Nginx, not by us.
See https://www.nginx.com/resources/wiki/extending/converting/#complex-example

Reply:
fix